tokens/style-dictionary.js

```diff
@@ -9,7 +9,13 @@ const cssUtilities = require('./css-utilities');
 const { formattedVariables, fileHeader, sortByReference } = StyleDictionary.formatHelpers;
 
 const colorTransform = (token) => {
-  const { value, modify = [] } = token;
+  const { value, modify = [], original } = token;
+  const reservedColorValues = ['inherit', 'initial', 'revert', 'unset'];
```

Comment:
[clarification] by default style-dictionary replaces such values with color values (e.g. `currentColor` would become `#000000`, see [related issue in style-dictionary](https://github.com/amzn/style-dictionary/issues/459)), this part of code allows us to skip such values without transforming them